factory/glideFactoryLib.py

```diff
@@ -1982,6 +1982,7 @@ def get_submit_environment(
                 ini_template = """[glidein_startup]
 args = %s
 idtoken_file_name = credential_frontend.idtoken
+proxy_file_name = pilot_proxy
 webbase= %s
 
 [vm_properties]
```

Comment:
Maybe change to use string interpolation:
```
                ini_template = f"""[glidein_startup]
args = {glidein_arguments}
idtoken_file_name = credential_frontend.idtoken
proxy_file_name = pilot_proxy
webbase= {web_url}
...
```

Reply:
reasonable suggestion but I did not implement, it works as is